toDoApp/migrations/0001_initial.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 5.0.7 on 2024-08-06 10:07
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
```

Comment:
this is django default user model . 
lets create a custom user model and use here